lbmerchant93/petstrology
# Feature remove saved/link signs

What does this PR do?
Clean up images code to be accessed whenever imported. Add remove from saved functionality. Link the found sign to the horoscope so they can visit right away. 

What does it feature/fix?
Feature remove searched sign functionality. Link found sign. Fix styling some with new background and color of text.

Where should the reviewer start?
FindSign.js line 12

How should this be tested?
Should be able to add some signs by searching and then removing whichever ones you want with clicking the remove button. Should be able to go to the Horoscope page for the sign that was found by clicking the link in the searched signs. 

What are the next steps or any issues/concerns?
Testing
Create footer information
